tutorial/source/mini_pyro.py

```diff
@@ -0,0 +1 @@
+../../examples/mini_pyro.py
```

Comment:
does this build with `make html`?

Reply:
oops no, I accidentally left a stray simlink; will delete...